package.json

```diff
@@ -56,14 +56,15 @@
 		"react": "^17.0.0",
 		"react-content-loader": "^6.2.0",
 		"react-dom": "^17.0.0",
-		"react-hook-form": "7.37.0",
+		"react-hook-form": "^7.43.1",
 		"react-i18next": "^11.7.2",
 		"react-json-view": "^1.21.1",
 		"react-qr-code": "^1.0.2",
 		"react-redux": "^7.2.0",
 		"react-router-dom": "^5.1.2",
 		"react-simple-tree-menu": "^1.1.18",
 		"reactstrap": "^8.4.1",
-		"redux": "^4.0.1"
+		"redux": "^4.0.1",
+		"terser-webpack-plugin": "^5.3.6"
```

Comment:
@petrKavulok Shouldnt be `terser-webpack-plugin` in devDependencies rather in `dependencies` ? `uglify` as in devDependencies as well - but anyway, this is on you to consider that, since you have a broader experience with webpack than I do